drivers/spi/Kconfig.andes_atcspi200

```diff
@@ -0,0 +1,12 @@
+# Kconfig Andes ATCSPI200 SPI configuration options
+#
+# Copyright (c) 2019 Andes Technology Corporation.
```

Comment:
s/2019/2022

Reply:
Thanks for reviewing.

I have fixed this incorrect date.